distributionviewer/core/static/js/app/components/views/chart.js

```diff
@@ -24,7 +24,8 @@ export class Chart extends React.Component {
   }
 
   componentWillMount() {
-    axios.get(`${metricApi.endpoints.GET_METRIC}${this.props.chartId}/`).then(response => {
+    const todayISO = new Date().toISOString().substring(0, 10);
+    axios.get(`${metricApi.endpoints.GET_METRIC}${this.props.chartId}/?date=${todayISO}`).then(response => {
```

Comment:
Technically this isn't required since the same thing will happen on the backend.